config/locales/react/react.de.yml

```
@@ -0,0 +1,3 @@
+de:
+  react:
+    your_footprint_result_text: Your annual climate footprint is %{footprint} %{unit_of_co2} of carbon dioxide. For %{price} %{time_unit} we help you climate compensate your imprint. The sum is distributed to different climate footprints that make a difference to the climate.
```

När det är en sån här röd varningssymbol så fattas en newline i EOF, så:
```suggestion
    your_footprint_result_text: Your annual climate footprint is %{footprint} %{unit_of_co2} of carbon dioxide. For %{price} %{time_unit} we help you climate compensate your imprint. The sum is distributed to different climate footprints that make a difference to the climate.
    
```